runtime/fundamentals/workspaces.md

```diff
@@ -383,3 +383,137 @@ root and its members:
 | version            | ❌        | ✅      |                                                                                                                                                                                                                 |
 | exports            | ❌        | ✅      |                                                                                                                                                                                                                 |
 | workspace          | ✅        | ❌      | Nested workspaces are not supported.                                                                                                                                                                            |
+
+## Running Commands Across Workspaces
+
+Deno provides several ways to run commands across all or specific workspace
+members:
+
+### Running Tests
+
+To run tests across all workspace members, simply execute `deno test` from the
+workspace root:
+
+```sh
+deno test
+```
+
+This will run tests in all workspace members according to their individual test
+configurations.
+
+To run tests for a specific workspace member, you can either:
+
+1. Change to that member's directory and run the test command:
+
+```sh
+cd my-directory
+deno test
+```
+
+2. Or specify the path from the workspace root:
+
+```sh
+deno test my-directory/
+```
+
+### Formatting and Linting
+
+Similar to testing, formatting and linting commands run across all workspace
+members by default:
+
+```sh
+deno fmt
+deno lint
+```
+
+Each workspace member follows its own formatting and linting rules as defined in
+its `deno.json` file, with some settings inherited from the root configuration
+as shown in the table above.
+
+### Using Workspace Tasks
+
+You can define tasks at both the workspace root and in individual workspace
+members:
+
+```json title="deno.json"
+{
+  "workspace": ["./add", "./subtract"],
+  "tasks": {
+    "build": "echo 'Building all packages'",
+    "test:all": "deno test"
+  }
+}
+```
+
+```json title="add/deno.json"
+{
+  "name": "@scope/add",
+  "version": "0.1.0",
+  "exports": "./mod.ts",
+  "tasks": {
+    "build": "echo 'Building add package'",
+    "test": "deno test"
+  }
+}
+```
+
+To run a task defined in a specific package:
+
+```sh
+deno task --cwd=add build
```

Comment:
I'm actually surprised this affects which task is resolved. I tried this out and it does work though.